src/lib/components/overlay/Overlay.svelte

```diff
@@ -41,7 +41,7 @@ function captureKeydown(event) {
     }
 }
 
-let classes = 'fixed top-0 left-0 right-0 w-screen h-screen grid place-content-center text-lg z-[120]'
+let classes = 'fixed top-0 left-0 right-0 w-screen h-screen grid place-content-center text-lg z-[120]  pointer-events-none'
```

Comment:
this is prep work that is needed to enable a drag and drop when an overlay covers the drop target.